tools/src/converter.ts

```diff
@@ -13,6 +17,19 @@ function convertTypes(types: ITypeDescr[]): string[] {
     return Array.from(convertedTypes);
 }
 
+function expandTypes(types: ITypeDescr[], customTypes: ICustomType[]): ITypeDescr[] {
+    let expandedTypes = [];
+    types.forEach((t) => {
+      if (t.isCustomType) {
+        const aliases = customTypes.filter((ct) => ct.name === t.type)[0].types;
+        if (aliases) {
+            expandedTypes = expandedTypes.concat(aliases);
+        }
+      }
+    });
+    return expandedTypes;
```

Comment:
What about this:
```
const expandedTypes = [];
...
expandedTypes.push(...aliases);
...
return expandedTypes;
```
It would be clearer, I think.